src/VisualStudio/Core/Next/Remote/ServiceHubRemoteHostClient.cs

```diff
@@ -73,28 +73,16 @@ internal sealed partial class ServiceHubRemoteHostClient : RemoteHostClient
                 return;
             }
 
-            // don't block UI thread while initialize workspace host
-            var host = new WorkspaceHost(vsWorkspace, client);
-
-            // Initialize the remote side with whatever data we have currently for the workspace.
-            // As workspace changes happen, this host will get notified, and it can remote those
-            // changes appropriately over to the remote size.
-            await host.InitializeAsync().ConfigureAwait(false);
```

Comment:
this event doesn't happen frequently, getting new session each time shouldn't be an issue.